accounts/models.py

```diff
@@ -67,6 +68,7 @@ def create_superuser(self, email, password=None, first_name=None, last_name=None
         # only set staff and admin to true
         user.staff = True
         user.admin = True
+        user.terms_of_use = True
```

Comment:
need this to be true for superusers to create organizations

Reply:
sounds good, I think during the super user sign up we can just let them know they automatically agree to our terms by becoming a super user